website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,185 @@
+tenants. Operators that deploy an `accept all traffic from anywhere in the cluster` style policy, are creating an obstacle in
+pursuing this goal, especially if it cannot be disabled. In security-conscious environments policies like these are not allowed
+in production. In such cases your operator should minimally **have the option to prevent [NetworkPolicy][k8s-network-policy] objects from being
+created** and leave these to the user. In more advanced cases, your operator should create [NetworkPolicy][k8s-network-policy] 
+configurations that follow the least-privilege principle, i.e. denying access to everything and from everywhere by default and only allowing access to specific authorized resources from specific authorized components.
+
+### Managing Resources
+
+#### Resource Requests
+
+Create resource requests for CPU and memory:
+
+* for your Operator's manager `Deployment` (See `config/manager/manager.yaml`)
+* for any `Pods` and `Deployments` which are created by your Operator
+
+```yml
+  ...
+  # TODO(user): Configure the resources accordingly based on the project requirements.
+  # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  resources:
+    requests:
+      cpu: 10m
+      memory: 64Mi
+    ... 
+```
+
+In order to prevent tenants from consuming all the resources on a cluster or blocking other workloads 
+from being scheduled, many production clusters will define [ResourceQuota][k8s-resources-quotas] configuration.
+[ResourceQuota][k8s-resources-quotas] configuration in a namespace will cause all pods without 
+[Requests or limits][k8s-requests-limits] for `CPU` and `Memory` to be rejected.
+
+Cluster administrators will naturally want to provide enough resource requests to their Operators in order to give them 
+sufficient memory resources for safe operations, and avoid that other workloads impact them negatively.
+For this reason, if you are integrating with [OLM][olm], you should, **always ship your operator `Deployment` 
+with resource requests amounts for CPU and memory**.
+
+The [ResourceQuota][k8s-resources-quotas] configuration also is applied to tenant workloads that are managed by your operator. Cluster administrators will typically set
+a [ResourceQuota][k8s-resources-quotas] for each tenant's namespace as part of the onboarding. If your operator creates `Pods/Deployments` inside
+the tenant namespace which do not specify at least resource requests for `CPU` and `Memory`, those `Pods/Deployments` may fail to schedule and tenants
+may not be able to use your operator. Therefore, your operator is scheduled, it should **always apply at least resource requests for CPU and
+memory** for `Pods/Deployments` that it creates as part of the reconciliation logic. Ideally your operator also applies 
+memory limits to those `Pods/Deployments`. You may also consider CPU limits.
+
```

Comment:
I see that LimitRange is mentioned below. I am not sure whether it would be beneficial to give a hint on it here.

Reply:
Not sure if I can follow up on your comment? 
What would be your suggestion here?

Reply:
I had something like that in mind:
"If your operator creates `Pods/Deployments` inside the tenant namespace..."  => "If a LimitRange  with default values has not been created in each namespace and  your operator creates `Pods/Deployments` inside the tenant namespace..."
I was not sure it is needed because we have later in the page:
"configurations made by the cluster administrators such as [ResourceQuota][k8s-resources-quotas] might not work without LimitRanges. The LimitRanger admission controller can provide default values for resource requests and limits when they have not been defined."

Reply:
anyway, I updated the text as sugested. Tks.